bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/MuteCommand.java

```diff
@@ -41,7 +41,7 @@ public MuteCommand() {
     @Override
     public void onExecute(NetworkCommandSender sender, String[] args) {
         if(BanSystem.getInstance().getConfig().banMode == BanMode.SELF){
-            if(!sender.hasPermission(reason.getPermission())&& !sender.hasPermission("dkbans.*")){
+            if(!sender.hasPermission(this.permission)&& !sender.hasPermission("dkbans.*")){
                 sender.sendMessage(Messages.REASON_NO_PERMISSION
                         .replace("[prefix]",getPrefix())
                         .replace("[reason]",reason.getDisplay()));
```

Comment:
reason variable is not available
this.permission will be null
Wrong permission message